<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>554</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>730</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Main</string>
     </attribute>
     <widget class="QPushButton" name="refreshButton">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>90</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
     <widget class="QLabel" name="numberOfBlocks">
      <property name="geometry">
       <rect>
        <x>534</x>
        <y>130</y>
        <width>472</width>
        <height>17</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>IBeamCursor</cursorShape>
      </property>
      <property name="text">
       <string>N/A</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>306</x>
        <y>197</y>
        <width>216</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Dice Info</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>286</x>
        <y>10</y>
        <width>20</width>
        <height>461</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="longestChain">
      <property name="geometry">
       <rect>
        <x>534</x>
        <y>150</y>
        <width>472</width>
        <height>17</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>IBeamCursor</cursorShape>
      </property>
      <property name="text">
       <string>N/A</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
     <widget class="QLabel" name="minBet">
      <property name="geometry">
       <rect>
        <x>534</x>
        <y>220</y>
        <width>472</width>
        <height>17</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>IBeamCursor</cursorShape>
      </property>
      <property name="text">
       <string>N/A</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>306</x>
        <y>150</y>
        <width>216</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Longest chain</string>
      </property>
     </widget>
     <widget class="QPushButton" name="refreshButton_2">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>180</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Info</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>306</x>
        <y>290</y>
        <width>411</width>
        <height>151</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>10</y>
         <width>261</width>
         <height>29</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayoutAmount" stretch="0">
        <item>
         <widget class="QLineEdit" name="betAmount"/>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="amountLabel">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>10</y>
         <width>64</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Amount:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>betAmount</cstring>
       </property>
      </widget>
      <widget class="QLabel" name="oddsLabel">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>40</y>
         <width>64</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>Odds:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>betAmount</cstring>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>40</y>
         <width>261</width>
         <height>29</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayoutAmount_4" stretch="0">
        <item>
         <widget class="QLineEdit" name="oddsAmount"/>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="betButton">
       <property name="geometry">
        <rect>
         <x>303</x>
         <y>80</y>
         <width>99</width>
         <height>27</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Bet</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>306</x>
        <y>107</y>
        <width>216</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Block chain</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>306</x>
        <y>260</y>
        <width>216</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Max odds</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="logWindow">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>30</y>
        <width>281</width>
        <height>391</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>306</x>
        <y>240</y>
        <width>216</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Max bet</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>306</x>
        <y>10</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Balance:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>306</x>
        <y>220</y>
        <width>216</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Min bet</string>
      </property>
     </widget>
     <widget class="QLabel" name="link1">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>425</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>BitcoinTalk</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="maxOdds">
      <property name="geometry">
       <rect>
        <x>534</x>
        <y>260</y>
        <width>472</width>
        <height>17</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>IBeamCursor</cursorShape>
      </property>
      <property name="text">
       <string>N/A</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
     <widget class="QLabel" name="maxBet">
      <property name="geometry">
       <rect>
        <x>534</x>
        <y>240</y>
        <width>472</width>
        <height>17</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>IBeamCursor</cursorShape>
      </property>
      <property name="text">
       <string>N/A</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>10</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Log window:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>306</x>
        <y>130</y>
        <width>216</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Current number of blocks</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdBalance">
      <property name="geometry">
       <rect>
        <x>306</x>
        <y>30</y>
        <width>411</width>
        <height>51</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="smallDecimalPoint">
       <bool>true</bool>
      </property>
      <property name="digitCount">
       <number>16</number>
      </property>
      <property name="mode">
       <enum>QLCDNumber::Dec</enum>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
      <property name="value" stdset="0">
       <double>0.000000000000000</double>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>History</string>
     </attribute>
     <widget class="QTableWidget" name="tableHistory">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>706</width>
        <height>391</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
